---
- hosts: localhost
  gather_facts: false
  tasks:
   - block:
      - name: ping google.com
        command: ping -c 2 google.com
        register: ping_result
        retries: 2
        delay: 5
     rescue:
       - name: Error occured
         debug:
           msg: "Error occured on {{ inventory_hostname }} {{ ansible_failed_task.name }}"

   - debug:
       msg: "Ping result of google.com {{ ping_result }}"